.circleci/cimodel/data/windows_build_definitions.py

```diff
@@ -123,18 +123,16 @@ def TruePred(_):
     WindowsJob(None, VcSpec(2017, ["14", "11"]), CudaVersion(10, 1), master_only_pred=FalsePred),
     WindowsJob(1, VcSpec(2017, ["14", "11"]), CudaVersion(10, 1)),
     # VS2017 no-CUDA (builds only)
-    WindowsJob(None, VcSpec(2017, ["14", "16"]), CudaVersion(10, 1)),
     WindowsJob(None, VcSpec(2017, ["14", "16"]), None),
     # VS2019 CUDA-10.1
     WindowsJob(None, VcSpec(2019), CudaVersion(10, 1)),
     WindowsJob(1, VcSpec(2019), CudaVersion(10, 1)),
     WindowsJob(2, VcSpec(2019), CudaVersion(10, 1)),
     # VS2019 CPU-only
     WindowsJob(None, VcSpec(2019), None),
-    WindowsJob(1, VcSpec(2019), None),
+    WindowsJob(1, VcSpec(2019), None, master_only_pred=TruePred),
     WindowsJob(2, VcSpec(2019), None, master_only_pred=TruePred),
-    WindowsJob(1, VcSpec(2019), CudaVersion(10, 1), force_on_cpu=True),
-    WindowsJob(2, VcSpec(2019), CudaVersion(10, 1), force_on_cpu=True, master_only_pred=TruePred),
+    WindowsJob(1, VcSpec(2019), CudaVersion(10, 1), force_on_cpu=True, master_only_pred=TruePred),
```

Comment:
What's going on here? (Loss of line 137 from original)

Reply:
This deletes vs2019_cuda_10.1_force_on_cpu_test2 (thus limiting force_on_cpu configuration to just smoke tests (test_nn) )